cognitive_architecture/classifiers/classifier.py

```diff
@@ -3,170 +3,180 @@
 from langchain.prompts import ChatPromptTemplate
 import json
 
-#TO DO, ADD ALL CLASSIFIERS HERE
-
-
-
-
-
+# TO DO, ADD ALL CLASSIFIERS HERE
 
 
 from langchain.chains import create_extraction_chain
 from langchain.chat_models import ChatOpenAI
 
 from ..config import Config
 from ..database.vectordb.loaders.loaders import _document_loader
+
 config = Config()
 config.load()
 OPENAI_API_KEY = config.openai_key
 from langchain.document_loaders import TextLoader
 from langchain.document_loaders import DirectoryLoader
 
 
-async def classify_documents(query:str, document_id:str, content:str):
-
-    document_context  = content
+async def classify_documents(query: str, document_id: str, content: str):
```

Comment:
Would be good to split those classifiers into multiple files, each containing one classifier. It will be easier to maintain.

Reply:
Refactored